images/Dockerfile.recovery-controller

```diff
@@ -0,0 +1,11 @@
+FROM golang:1.14-alpine AS build-env
+ADD ./recovery /recovery
+WORKDIR /recovery
+RUN apk add --no-cache git
+ENV GO111MODULE=on
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o cassandra-recovery
+
+FROM debian:buster-20200422-slim
```

Comment:
Why not `scratch`? In case we need a file system, let's use `debian:buster-slim` instead, because fixing this to a specific version also means that there won't be any security fixes to the tooling if we would rebuild this image. And rebuilding with an updated `debian:buster-slim` won't break anything, because the binary doesn't depend on the file system.

Reply:
Because I failed to make it work with scratch. Figured it out now, there's a subtle but important different between:
```
CMD /cassandra-recovery/cassandra-recovery
```
and
```
CMD [ "/cassandra-recovery/cassandra-recover" ]
```

The former requires bash to be installed...

Changed it to scratch